jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -41,6 +41,13 @@
       become: true
       when: sshd_config.changed
 
+    - name: Shrink size of syslog
+      become: true
+      become_user: root
+      shell:
+        cmd: truncate --size 10737418240 syslog.1 && truncate --size 10737418240 syslog  2>/dev/null
```

Comment:
Using `truncate` is exactly right for this task. However, I don't this will work without the full path to syslog (/var/log/syslog). Otherwise how do we know where the syslog file is? This makes me think that this change has not been manually tested. Could you try running the playbook and let me know if the task runs and if there are any changes needed? (Also - what if syslog.1 doesn't exist? It doesn't have to, maybe we don't have log rotation set up, or maybe the log hasn't gotten big enough to be rotated. Perhaps we should have some tolerance of failure on this command.)

Reply:
I tested a playbook after addressing your comments. It is working.  Is the following filed enough for failure tolerance? 
ignore_errors: true